src/components/TodoItem.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
+import { User } from './User';
+
+interface TodoProps {
```

Comment:
`ComponentName` + `Props`
```suggestion
interface TodoItemProps {
```

or make it easy
```suggestion
interface Props {
```